mapie/conformity_scores/sets/topk.py

```diff
@@ -0,0 +1,227 @@
+from typing import Optional, Union, cast
+
+import numpy as np
+
+from mapie.conformity_scores.classification import BaseClassificationScore
+from mapie.conformity_scores.sets.utils import (
+    check_proba_normalized, get_true_label_position
+)
+from mapie.estimator.classifier import EnsembleClassifier
+
+from mapie._machine_precision import EPSILON
+from mapie._typing import NDArray
+from mapie.utils import compute_quantiles
+
+
+class TopK(BaseClassificationScore):
+    """
+    Top-K method-based non-conformity score.
+
+    It is based on the sorted index of the probability of the true label in the
+    softmax outputs, on the calibration set. In case two probabilities are
+    equal, both are taken, thus, the size of some prediction sets may be
+    different from the others.
+
+    References
+    ----------
+    [1] Anastasios Nikolas Angelopoulos, Stephen Bates, Michael Jordan
+    and Jitendra Malik.
+    "Uncertainty Sets for Image Classifiers using Conformal Prediction."
+    International Conference on Learning Representations 2021.
+
+    Attributes
+    ----------
+    classes: Optional[ArrayLike]
+        Names of the classes.
+
+    random_state: Optional[Union[int, RandomState]]
+        Pseudo random number generator state.
+
+    quantiles_: ArrayLike of shape (n_alpha)
+        The quantiles estimated from ``get_sets`` method.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+    def set_external_attributes(
```

Comment:
same as for LAC : is it needed ?

Reply:
If in the future we need to redefine it, then we will rewrite it. However, I think to make the code more concise, we should remove it.